### ISSUE TITLE
Records Computer updates DNA Scrambler User's Name, but not DNA or Prints

### ISSUE THREAD

**Issue by EquanimityX0880** [DeltaV-Station/Delta-v#2178]
## Description

This issue was discovered on Apoapsis and tested on my local server. The DNA scrambler physically changes name, appearance, DNA sequence, and fingerprint pattern. However, the Station and Criminal Records Computer updates this new character's **name**, rendering the scrambler useless in avoiding suspicions based on blood or fingerprint evidence.

**Reproduction**

Record your character's original DNA and Fingerprint sequence. Use a Scrambler. Check Station records.

**Screenshots**

![image](https://github.com/user-attachments/assets/2394ba64-d425-45ec-887e-45c94ffd0bfb)
_Before - > After scramble_

**Additional context**

NA

**Comment by deltanedas** [comment]
it shouldnt update records in the first place, if theres say only 1 salv and the salv magically changes name then you just know they scrambled

**Comment by MilonPL** [comment]
> it shouldnt update records in the first place, if theres say only 1 salv and the salv magically changes name then you just know they scrambled

do you think it should just remove the record instead? or leave it as is?

**Comment by EquanimityX0880** [comment]
I agree. A Detective shouldn't be able to use DNA evidence to link a crime to someone who used a DNA scrambler. 

... Which is exactly what happened in a recent Apoapsis round. A player left blood evidence, scrambled their DNA to RP as a Central Command agent, then had the Detective close in on 'em for the rest of the round. The player who discovered this likely won't try this elaborate, emergent RP again - Which is why wanted to document it.

**Comment by EquanimityX0880** [comment]
> do you think it should just remove the record instead? or leave it as is?

I don't believe that a Fugitive's information appears in station records.. I think it would be best to delete the the original entry the DNA Scrambler user had. The records are deleted when the player heads to Cryo, right?
